sync-repos.sh

```diff
@@ -44,7 +46,7 @@ if [ -n "${GENIE_VERSION}" ] && [ "${GENIE_VERSION}" != "${GENIE_HEAD}" ]; then
   git fetch
   git checkout -f ${GENIE_VERSION}
   if test -f yarn.lock ; then
-    yarn link thingtalk
+#    yarn link thingtalk
```

Comment:
This line is wrong, and also should have no effect on recent versions of genie that have switched to npm. On versions of genie that use yarn, it's necessary or nothing works.